src/cmdstan/write_profiling.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef CMDSTAN_WRITE_PROFILING_HPP
+#define CMDSTAN_WRITE_PROFILING_HPP
+
+#include <stan/math/rev/core/profiling.hpp>
+#include <stan/callbacks/writer.hpp>
+#include <stan/version.hpp>
+#include <string>
+#include <vector>
+
+namespace cmdstan {
+void write_profiling(stan::callbacks::writer& writer,
+                     stan::math::profile_map& p) {
+  stan::math::profile_map::iterator it;
+  std::stringstream profile_csv_stream;
+  profile_csv_stream << "name,thread_id,time,forward_time,reverse_time,chain_"
```

Comment:
I reverse my statement. `total_time` or `time_total` is better.

Reply:
total_time, since its consistent with forward_time and reverse_time